openmc/deplete/microxs.py

```diff
@@ -327,7 +329,7 @@ def from_multigroup_flux(
                         xs = lib_nuc.collapse_rate(
                             mt, temperature, energies, multigroup_flux
                         )
-                        microxs_arr[nuc_index, mt_index] = xs
+                        microxs_arr[nuc_index, mt_index, 0] = xs
 
         return cls(microxs_arr, nuclides, reactions)
```

Comment:
I think the changes above look good, but would this also do the trick?

```suggestion
        # ensure microxs_arr has the correct number of dimensions for a one-group cross section
        microxs_arr = np.expand_dims(microxs_arr, -1)
        return cls(microxs_arr, nuclides, reactions)
```

Reply:
That would work if we keep `microxs_arr` as 2D, but it is a lot less obvious to someone reading the code in my opinion, so I'd prefer the current version (explicit is better than implicit).